src/test/java/uk/gov/ons/ctp/integration/envoycuc/client/RateLimitClient.java

```diff
@@ -1,12 +1,13 @@
 package uk.gov.ons.ctp.integration.envoycuc.client;
 
+import org.springframework.cloud.client.circuitbreaker.CircuitBreaker;
 import uk.gov.ons.ctp.common.rest.RestClient;
 import uk.gov.ons.ctp.integration.ratelimiter.client.RateLimiterClient;
 
 public class RateLimitClient extends RateLimiterClient implements TestClient {
```

Comment:
Comment in passing as I see that the class already exists. 
Didn't know it was possible to have the classname the same as its superclass!! A little confusing.
Oh well, never mind

Reply:
The class name is different - I extended the class in order to implement an interface so that I could use the same code for aither client type